browser/src/control/Control.Mention.ts

```diff
@@ -15,6 +15,7 @@
 /* global app */
 
 interface MentionUserData {
+	label: string;
```

Comment:
As with the comment below, maybe for backward compatibility this could be nullable